stackstate-deps.json

```diff
@@ -1,5 +1,5 @@
 {
-  "STACKSTATE_INTEGRATIONS_VERSION": "1.7.0",
+  "STACKSTATE_INTEGRATIONS_VERSION": "STAC-11119-service-name",
```

Comment:
Changes I tested with new molecule test are on `STAC-11119-service-name` branch.
So first we need to merge agent-integrations branch `STAC-11119-service-name` to `master` and do `1.8.0` release of agent-integrations. Then change it here, otherwise new nagios events molecule test will fail.